tests/spread/extensions/snaps/env-injector-hello/hello.c

```diff
@@ -4,34 +4,66 @@
 
 int main(int argc, char const *argv[]) {
     const char *envs[] = {
-        "HELLO",        // This is set globally
-        "WORLD",        // This is set for the app
-        "HELLO_WORLD",  // This is set from env file
+        "GLOBAL",       // This is set globally
+        "HELLO",        // This is set for the app
+        "HELLO_WORLD",  // This is set from global env file
     };
 
     const char *expected[] = {
-        "Hello",
         "World",
+        "Hello",
         "Hello World",
     };
 
     int num_envs = sizeof(envs) / sizeof(envs[0]);
 
+    // Check Global, App specific and global env file
     for (int i = 0; i < num_envs; ++i) {
         const char *env = getenv(envs[i]);
 
         if (env == NULL) {
             fprintf(stderr, "\n[ERROR] Env. variable %s is not set.\n", envs[i]);
-            return 1;
+            exit(1);
         }
 
         if (strcmp(env, expected[i]) != 0) {
             fprintf(stderr, "\n[ERROR] Env. variable %s isn't set to the expected value.\n", envs[i]);
             fprintf(stderr, "Expected: %s\n", expected[i]);
             fprintf(stderr, "Got: %s\n", env);
-            return 1;
+            exit(1);
         }
     }
 
+    // Check that it's not possible to access other app ENVs 
+    const char *env_specific = getenv("SPECIFIC");
+    if (env_specific != NULL) {
+        fprintf(stderr, "\n[ERROR] Env. variable SPECIFIC is accessible from the app.\n");
```

Comment:
Can't use [println](https://doc.rust-lang.org/std/macro.println.html)?

Reply:
But this is **C** code and not **Rust**!